pipeline/pipeline_cron.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+source activate datasci-sba
+cd $PYTHONPATH
+
+/Users/michaelmathews/anaconda3/bin/python3 pipeline/pipeline_api_controller.py "$@"
```

Comment:
Wonder if there's a way to make this general instead of `/Users/michaelmatthews`?

Reply:
The path in cron is very basic, but if we can set it correctly then you wouldn't need a user directory. I would expect on most production systems python would be installed somewhere other than a user directory.